src/renderers/DomRenderer.js

```diff
@@ -248,6 +248,23 @@ function insertStyle(style: HTMLElement, options: PriorityOptions) {
   getHead().insertBefore(style, prevNode)
 }
 
+/**
+ * Read jss settings from the page if the user has set them.
+ */
+function getSettingsFromInlineJSON() : Object {
+  let config = {}
+  const inlineJsonBlock = document.querySelector('script[type="application/json"][data-jss-selector="jss-settings-json"]')
+  if (inlineJsonBlock) {
+    try {
+      config = JSON.parse(inlineJsonBlock.textContent)
+    }
+    catch (error) {
+      warning('Error parsing jss-settings-json', error)
```

Comment:
where is this warning from? mb for the sake of example console.warn?

Reply:
I was going to use `console` but the linter complained about that, so I looked at the other code in the file and copied what I saw at https://github.com/cssinjs/jss/blob/master/src/renderers/DomRenderer.js#L244